go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/smartcontractkit/chainlink-ccip
 go 1.23.3
 
 require (
+	github.com/charmbracelet/lipgloss v1.0.0
```

Comment:
Can we have a separate go.mod for the tools?

Reply:
Would give us a bit more freedom

Reply:
done